lib/src/chart/bar_chart/bar_chart.dart

```diff
@@ -202,21 +202,27 @@ class _BarChartState extends AnimatedWidgetBaseState<BarChart> {
         touchResponse.touchInput is FlPanMoveUpdate ||
         touchResponse.touchInput is FlLongPressStart ||
         touchResponse.touchInput is FlLongPressMoveUpdate) {
-      setState(() {
+
+      if (!widget.data.barTouchData.stickyTouchIndicator) {
         if (touchResponse.spot == null) {
           _showingTouchedTooltips.clear();
           return;
         }
-        final groupIndex = touchResponse.spot.touchedBarGroupIndex;
-        final rodIndex = touchResponse.spot.touchedRodDataIndex;
-
-        _showingTouchedTooltips.clear();
+      } else {
+         if (touchResponse.spot == null) {
+          return;
+        }
+      }
+      final groupIndex = touchResponse.spot.touchedBarGroupIndex;
+      final rodIndex = touchResponse.spot.touchedRodDataIndex;
+      _showingTouchedTooltips.clear();
+      setState(() {
         _showingTouchedTooltips[groupIndex] = [rodIndex];
       });
     } else {
-      setState(() {
+      if (!widget.data.barTouchData.stickyTouchIndicator) {
         _showingTouchedTooltips.clear();
```

Comment:
It should be wrapped by `setState()` like previous.

Reply:
setstate didn't do anything here

Reply:
Okay, I will test it again, 
Thanks!